src/cfg/builder.rs

```diff
@@ -33,6 +36,10 @@ impl CfgBuilder {
         instructions: &[HbcFunctionInstruction],
         function_index: u32,
     ) -> DiGraph<Block, EdgeKind> {
+        // Clear lookup tables for a fresh build
```

Comment:
I feel like function_index should actually be passed into new(), as PC is scoped to the current function--there are no jumps between functions.  this should negate the need for clearing the PC/block starts each time build_from_instructions is called?